daemon/daemon.go

```diff
@@ -1011,10 +1011,20 @@ func (daemon *Daemon) Graph() *graph.Graph {
 }
 
 // Repositories returns all repositories.
+//
+// FIXME: remove this when there are no references left (after
+// gh#16638 & gh#16709 are both merged)
```

Comment:
I believe both are merged now so you should be able to remove this as part of this PR.